tools/car_porting/test_car_model.py

```diff
@@ -1,38 +1,20 @@
 #!/usr/bin/env python3
-import argparse
-import sys
-import unittest # noqa: TID251
+import os
+from typing import Any
 
 from openpilot.selfdrive.car.tests.routes import CarTestRoute
-from openpilot.selfdrive.car.tests.test_models import TestCarModel
+from openpilot.selfdrive.car.tests.test_models import TestCarModelBase
 from openpilot.tools.lib.route import SegmentName
 
+route = os.getenv('ROUTE')
+car: Any = os.getenv('CAR')
 
-def create_test_models_suite(routes: list[CarTestRoute], ci=False) -> unittest.TestSuite:
-  test_suite = unittest.TestSuite()
-  for test_route in routes:
-    # create new test case and discover tests
-    test_case_args = {"platform": test_route.car_model, "test_route": test_route, "test_route_on_bucket": ci}
-    CarModelTestCase = type("CarModelTestCase", (TestCarModel,), test_case_args)
-    test_suite.addTest(unittest.TestLoader().loadTestsFromTestCase(CarModelTestCase))
-  return test_suite
+assert route is not None, "Must specify route or segment name"
 
+route_or_segment_name = SegmentName(route, allow_route_name=True)
+segment_num = route_or_segment_name.segment_num if route_or_segment_name.segment_num != -1 else None
+test_route = CarTestRoute(route_or_segment_name.route_name.canonical_name, car, segment=segment_num)
```

Comment:
all this stuff runs at import time now. it should only be run if we're running this script